ush/rocoto/setup_workflow_fcstonly.py

```diff
@@ -299,40 +320,68 @@ def get_workflow(dict_configs, cdump='gdas'):
     else:
         if hpssarch in ['YES']:
             deps = []
-            data = '&ROTDIR;/&CDUMP;.@Y@m@d/@H/atmos/INPUT/sfc_data.tile6.nc'
+            if cdump == "gefs":
```

Comment:
To match the normal python idiom (and the rest of the code), these should all be `if cdump in ['gefs'].